zcash_test_vectors/transaction_v6.py

```diff
@@ -30,7 +32,8 @@ def __init__(self, rand):
         isk = IssuanceKeys(rand.b(32))
         desc_size = rand.u32() % 512 + 1
         desc_bytes = get_random_unicode_bytes(desc_size, rand)
-        asset_digest_bytes = asset_digest(encode_asset_id(isk.ik_encoding, desc_bytes))
+        asset_desc_hash = asset_desc_digest(desc_bytes)
+        asset_digest_bytes = asset_digest(encode_asset_id(isk.ik_encoding, asset_desc_hash))
```

Comment:
move `encode_asset_id()` to it's own line

Reply:
Done